crates/test-util/src/testcontainers/mod.rs

```diff
@@ -1,4 +1,6 @@
-pub use testcontainers::{core::Mount, runners::AsyncRunner, ContainerAsync, ImageExt};
+pub use testcontainers::{
+    core::ExecCommand, core::Mount, runners::AsyncRunner, ContainerAsync, ImageExt,
```

Comment:
nit; I like to split my imports per module, sort of "Go style" with multiple lines, but could we at least merge the ones from `core` into a single block?
Your LSP should be able to do the merge here. (I'd then also split `core` imports using LSP, but that's up to you)

Reply:
I'm probably missing some `rustfmt`(?) configuration to make this the default, I'd love to automate this but I'm surely missing something in my editor setup.